spec/controllers/suggested_councillors_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
This call to a `rails_helper` file that we don't have is breaking the tests suit. This is another file created by the rails generator. I think just remove the whole file until we want to write controller tests for this.